Tests/Weaver/Realm.Fody.Tests/TestHelpers.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ////////////////////////////////////////////////////////////////////////////
+// //
+// // Copyright 2021 Realm Inc.
+// //
+// // Licensed under the Apache License, Version 2.0 (the "License")
+// // you may not use this file except in compliance with the License.
+// // You may obtain a copy of the License at
+// //
+// // http://www.apache.org/licenses/LICENSE-2.0
+// //
+// // Unless required by applicable law or agreed to in writing, software
+// // distributed under the License is distributed on an "AS IS" BASIS,
+// // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// // See the License for the specific language governing permissions and
+// // limitations under the License.
+// //
+// ////////////////////////////////////////////////////////////////////////////
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.XPath;
+using System.Xml.Xsl;
+
+namespace Realms.Fody.Tests
+{
+    public static class TestHelpers
+    {
+        public static void TransformTestResults(string resultPath)
```

Comment:
Rather than copy these methods, let's split TestHelpers into two partial classes and have the weaver reference just one part.